File: trunk/StudMap/StudMap.Core/Maps/Floor.cs
using System;
namespace StudMap.Core.Maps
{
    public class Floor
    {
        public int Id { get; set; }

        public int MapId { get; set; }

        public string Name { get; set; }

        public string ImageUrl { get; set; }

        public DateTime CreationTime { get; set; }
    }
}

File: StudMap.Navigator/src/de/whs/studmap/data/Floor.java
package de.whs.studmap.data;

public class Floor {

	private int id;
	private int mapId;
	private String imageUrl;
	private String name;
	
	public Floor(int id, int mapId, String url, String name) {
		this.id = id;
		this.mapId = mapId;
		this.imageUrl = url;
		this.name = name;
	}
	
	public int getId() {
		return id;
	}
	
	public int getMapId() {
		return mapId;
	}
	public String getUrl() {
		return imageUrl;
	}
	public String getName() {
		return name;
	}
	
	@Override
	public String toString(){
		return name;
	}
}
